packages/manager/modules/pci/src/projects/project/storages/containers/users/delete/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "pci_projects_project_storages_containers_users_delete_title": "Supprimer utilisateur",
+  "pci_projects_project_storages_containers_users_delete_label": "Supprimer",
+  "pci_projects_project_storages_containers_users_delete_question": "Etes-vous sur de supprimer cet utilisateur des utilisateurs S3 Policies ?",
+  "pci_projects_project_storages_containers_users_delete_success": "L’utilisateur {{ user }} a bien été supprimé de la liste des utilisateurs S3.",
+  "pci_projects_project_storages_containers_users_delete_error": "Une erreur s'est produite lors de la suppression de l'utilisateur: {{message}}"
```

Comment:
This error message need to be corrected to handle partial delete success. Few users might be deleted.